packages/ceramic-doctype-three-id/src/three-id-doctype.ts

```diff
@@ -50,17 +55,15 @@ export class ThreeIdDoctype extends Doctype {
     /**
      * Creates genesis record
      * @param params - Create parameters
-     * @param context - Ceramic context
-     * @param opts - Initialization options
      */
-    static async makeGenesis(params: Record<string, any>, context?: Context, opts: DocOpts = {}): Promise<Record<string, any>> {
-        const { content, owners } = params
+    static async makeGenesis(params: Record<string, any>): Promise<Record<string, any>> {
+        const { content, metadata } = params
 
-        if (!owners) {
+        if (!metadata?.owners) {
             throw new Error('The owner of the 3ID needs to be specified')
         }
         return {
-            doctype: DOCTYPE, owners, content
+            doctype: DOCTYPE, metadata, content
```

Comment:
To avoid confusion, the metadata should be put into the `header` of records: https://github.com/ceramicnetwork/specs#document-records

Reply:
I'll refactor it, tnx!